bip-0148.mediawiki

```diff
@@ -24,7 +24,9 @@ This document specifies a BIP16 like soft fork flag day activation of the segreg
 
 Segwit increases the blocksize, fixes transaction malleability, and makes scripting easier to upgrade as well as bringing many other [https://bitcoincore.org/en/2016/01/26/segwit-benefits/ benefits].
 
-It is hoped that miners will respond to this BIP by activating segwit early, before this BIP takes effect. Otherwise this BIP will cause the mandatory activation of the existing segwit deployment before the end of midnight November 15th 2017.
+It is hoped that miners will respond to this BIP by activating segwit early, before this BIP takes effect. Otherwise this BIP may cause the mandatory activation of the existing segwit deployment before the end of midnight November 15th 2017.
+
+In the cases that the hashrate drops to below 14% over 3.5 months sustained, it is possible that SegWit is not locked-in before November 15th 2017.
```

Comment:
If the hashrate drops below 14% for a sustained 3.5 month period, it is possible that segwit will not be locked-in before November 15th 2017.